pkgs/by-name/li/libzeep/package.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  boost,
+  date,
+  mrc,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "libzeep";
+  version = "6.0.13";
+
+  src = fetchFromGitHub {
+    owner = "mhekkel";
+    repo = "libzeep";
+    rev = "refs/tags/v${finalAttrs.version}";
+    hash = "sha256-74mYjz/jXzEU3j1kHSvTvZcIfX9csVJKTKONAZXUMvs=";
+  };
+
+  prePatch = ''
```

Comment:
This should also be postPatch

Also what tries the sed to achieve?

Reply:
https://github.com/NixOS/nixpkgs/pull/328849#discussion_r1724149382

Reply:
It allows for non-exact versions of dependencies to be used.